### PR TITLE
Chal 697/fix firefox calendar scheduler

### PR DESCRIPTION
**Pull request by Homerade**
- [ ] README is up to date
- [ ] Docs are up to date with changes (modules and functions)
- [ ] Tests are included for changes
- [ ] Links in emails use the `_url` route helpers
- [ ] Controllers modified contain appropriate authorization plugs

### CODE REVIEW

File: assets/js/app/_datetime_field.js [hunk 1]
```diff
@@ -14,4 +14,38 @@ $("body").on("input", ".js-datetime-input", function() {
...
+  setCombinedDatetimeValue($(this))
+})
+
+$("body").on("input", ".js-time-input", function() {
+  setCombinedDatetimeValue($(this))
+})
+
+function setCombinedDatetimeValue(input) {
+  let date = $(".js-date-input").val()
+  let time = $(".js-time-input").val()
```
Comment (cpreisinger) [inline assets/js/app/_datetime_field.js:47]: These might have a problem if multiple `.js-date-input` and `.js-time-input` fields exist on the page. Probably need to find the specific sibling of the relevant classes from the `input` passed in

File: assets/js/app/_datetime_field.js [hunk 2]
```diff
@@ -14,4 +14,38 @@ $("body").on("input", ".js-datetime-input", function() {
...
+$(".js-date-input").each(function() {
+  let utc_time = $(this).siblings("input[type=hidden]").val()
+  let local_date = moment(utc_time).local().format("YYYY-MM-DD")
+
+  $(this).val(local_date)
+})
+
+$(".js-time-input").each(function() {
+  let utc_time = $(this).siblings("input[type=hidden]").val()
+  let local_time = moment(utc_time).local().format("HH:mm")
```
Comment (cpreisinger) [inline assets/js/app/_datetime_field.js:31]: Does this need AM/PM as well?

**Review by cpreisinger — CHANGES_REQUESTED** [review]

File: assets/js/app/_datetime_field.js (hunk 2 above)
Comment (Homerade) [reply assets/js/app/_datetime_field.js:31]: No. It takes 24h and converts for the UI

File: assets/js/app/_datetime_field.js (hunk 1 above)
Comment (Homerade) [reply assets/js/app/_datetime_field.js]:
```
  let date =
    Object.values(input[0].classList).includes("js-date-input")
      ? input.val()
      : input.siblings(".js-date-input").val()
  let time =
    Object.values(input[0].classList).includes("js-time-input")
      ? input.val()
      : input.siblings(".js-time-input").val()
```

**Pull request merged**